### ISSUE TITLE
Event listeners need to be bound

### ISSUE THREAD

**Issue by mlereste** [bambora/na-customcheckout#68]
Hello,

Maybe we have a wrong understanding of how this lib works, but here is an issue we encountered. Here are the steps to reproduce:

- Reproduce/copy the demo.html
- When you type in a field (eg: credit card number), if you type something incorrect the field border becomes red
- Remove the event listener `checkoutFields.on('error', function(event) { ... });`
- When you type in a field (eg: credit card number), if you type something incorrect the field border do not becomes red

Bottom line is: you need to add an event listener (even if the callback function is empty), otherwise the css error/completed/focus/etc classes are not applied.

Is that a normal/expected behavior?

**Comment by e-wipond** [comment]
Hi, thanks for the feedback. You're right, it's a bug. The expected behaviour is that the wrapper classes should be toggled without needing to have any event listeners set up. It's been fixed for the next release. 

A workaround in the meantime, as you mentioned, is just to set up empty (or non-empty) event listeners for the event types, and then you will get the expected behaviour.

**Comment by dotboris** [comment]
@e-wipond when you say

> It's been fixed for the next release.

Do you mean that the fix should be in the `dev` branch already?

**Comment by e-wipond** [comment]
It has not been pulled into the dev branch yet.

**Comment by e-wipond** [comment]
This has been fixed on the dev branch (now deployed at `https://s3.ca-central-1.amazonaws.com/na-customcheckout.dev/2.0.0/customcheckout.js`). It should make it's way to a production deploy in the coming days.
